Assets/UI/Scripts/Main/ExitGame.gd

```diff
@@ -1,7 +1,6 @@
 extends BaseButton
 
-var scene = load("res://Assets/UI/Scenes/ExitScene.tscn") # : PackedScene
+var _scene = preload("res://Assets/UI/Scenes/ExitScene.tscn") # : PackedScene
 
 func _pressed():
-	var _stfu = get_tree().change_scene_to(scene)
```

Comment:
Are we even gonna use this variable for anything? Because this method can be called directly.

Reply:
Godot 3.1 gives warnings about an unused return value from change_scene_to

Reply:
Then `#warning-ignore:return_value_discarded` should be used. Makes more sense than creating a variable just to silence the warning.

Reply:
And with that, another thing that somehow I didn't noticed before: Why each button has its own script? The menu scene should have a script that holds all of these methods.

Reply:
> #warning-ignore:return_value_discarded is longer to write than var _stfu = and does the same thing.

That's still creating a variable just to silence a warning! Besides being far more explicit, if there's a system to properly ignore warnings, that should be used instead of this.

> I'd suggest asking @LinuxDonald what his preferred programming paradigm is.

Considering that he's just starting to learn Godot as far as I'm aware, I don't think that would help that much, but I could give it a shot.
However, most projects that I've seen (including official ones: [1](https://github.com/godotengine/godot-demo-projects/blob/master/2d/dodge_the_creeps/Main.gd#L9), [2](https://github.com/godotengine/godot-demo-projects/blob/master/2d/dodge_the_creeps/Main.tscn#L110)) just connect the buttons to the main script, as it's much more manageable than hopping through scripts for such small tasks.

Reply:
Fair enough.